CHANGELOG.md

```diff
@@ -1,9 +1,19 @@
 # Changelog
```

Comment:
Don't forget to add `[2.2.0]: https://github.com/onfido/onfido-sdk-ui/compare/2.1.0...2.2.0` at the end of CHANGELOG.md